.cirrus.yml

```diff
@@ -70,10 +70,10 @@ task:
     QEMU_USER_CMD: ""  # Disable qemu and run the test natively
 
 task:
-  name: 'Win64 [unit tests, no gui tests, no boost::process, no functional tests] [jammy]'
+  name: 'Win64 [unit tests, no gui tests, no boost::process, no functional tests] [noble]'
```

Comment:
747c6d72bee5cf0359ee2820adbb927ce05bd45d: I think we shouldn't touch `.cirrus.yml` - we don't use it and changing it would simply cause merge conflicts later

Reply:
Reverted changes to `.cirrus.yml` and updated container base on some glossed over containers in latest push